storage/rocksdb/rdb_datadic.cc

```diff
@@ -890,8 +941,20 @@ uint Rdb_key_def::pack_record(const TABLE *const tbl, uchar *const pack_buffer,
     unpack_len_pos = unpack_info->get_current_pos();
     // we don't know the total length yet, so write a zero
     unpack_info->write_uint16(0);
+
+    if (m_index_type == INDEX_TYPE_SECONDARY) {
+      // Reserve two bytes for the covered bitmap. This will store for each key
+      // part whether or not the column is covered. A max of 16 columns are
+      // permitted for indexes.
+      covered_bitmap_pos = unpack_info->get_current_pos();
+      unpack_info->write_uint16(0);
```

Comment:
I bumped the sk format version and added checks for m_kv_format_version everywhere the bitmap is being read. Is that sufficient, or should we also check the version when writing?